qiskit/test/mock/utils/fake_backend_builder.py

```diff
@@ -0,0 +1,331 @@
+                                     Command, UchannelLO)
+from qiskit.providers.models.backendproperties import Nduv, Gate
+from qiskit.qobj import PulseQobjInstruction
+from qiskit.test.mock.fake_backend import FakeBackend
+
+
+class FakeBackendBuilder:
+    """FakeBackend builder.
+
+    For example:
+        backend = FakeBackendBuilder("Tashkent", n_qubits=100).build()
+    """
+
+    def __init__(self,
+                 name: str,
+                 n_qubits: int,
+                 version: Optional[str] = None,
+                 coupling_map: Optional[List[List[int]]] = None,
+                 basis_gates: Optional[List[str]] = None,
+                 qubit_t1: Optional[float] = None,
+                 qubit_t2: Optional[float] = None,
+                 qubit_frequency: Optional[float] = None,
+                 qubit_readout_error: Optional[float] = None,
+                 single_qubit_gates: Optional[List[str]] = None):
+        """Creates fake backend builder.
+
+        Args:
+            name (str): Name of the backend.
+            n_qubits (int): Number of qubits in the backend.
+            version (str, optional): Version of the fake backend.
+            coupling_map (list, optional): Coupling map.
+            basis_gates (list, optional): Basis gates of the backend.
+            qubit_t1 (float, optional): Longitudinal coherence time.
+            qubit_t2 (float, optional): Transverse coherence time.
+            qubit_frequency (float, optional): Frequency of qubit.
+            qubit_readout_error (float, optional): Readout error of qubit.
+            single_qubit_gates (list, optional): List of single qubit gates for backend properties.
+        """
+        if version is None:
+            version = '0.0.0'
+
+        if basis_gates is None:
+            basis_gates = ['id', 'u1', 'u2', 'u3', 'cx']
+
+        if qubit_t1 is None:
+            qubit_t1 = 113.3
+
+        if qubit_t2 is None:
+            qubit_t2 = 150.2
+
+        if qubit_frequency is None:
+            qubit_frequency = 4.8
+
+        if qubit_readout_error is None:
+            qubit_readout_error = 0.04
+
+        if single_qubit_gates is None:
+            single_qubit_gates = ['id', 'u1', 'u2', 'u3']
+
+        self.name = name
+        self.version = version
+        self.basis_gates = basis_gates
+        self.qubit_t1 = qubit_t1
+        self.qubit_t2 = qubit_t2
+        self.qubit_frequency = qubit_frequency
+        self.qubit_readout_error = qubit_readout_error
+        self.n_qubits = n_qubits
+        self.single_qubit_gates = single_qubit_gates
+        self.coupling_map = coupling_map
+        self.now = datetime.now()
+
+    @property
+    def cmap(self):
+        """Returns cmap of provided else generated Almadel like cmap."""
+        return self.coupling_map if self.coupling_map else self._generate_cmap()
+
+    def _generate_cmap(self) -> List[List[int]]:
+        """Generate default grid-like coupling map."""
```

Comment:
Is there a reason to just not use `CouplingMap.from_grid`? https://qiskit.org/documentation/stubs/qiskit.transpiler.CouplingMap.from_grid.html#qiskit.transpiler.CouplingMap.from_grid

Reply:
I think `from_grid` will not work for 99 qubits. Or it's ok to have dangling edges?